cpp/src/Ice/AddDefaultPluginFactories_min.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) ZeroC, Inc.
+
+#include "AddDefaultPluginFactories.h"
+
+using namespace std;
+
+extern "C"
+{
+    Ice::Plugin* createIceTCP(const Ice::CommunicatorPtr&, const std::string&, const Ice::StringSeq&);
+    Ice::Plugin* createIceSSL(const Ice::CommunicatorPtr&, const std::string&, const Ice::StringSeq&);
+}
+
+void
+IceInternal::addDefaultPluginFactories(vector<Ice::PluginFactory>& pluginFactories)
+{
+    pluginFactories.insert(pluginFactories.begin(), {{"IceTCP", createIceTCP}, {"IceSSL", createIceSSL}});
+}
+
+bool
+IceInternal::isMinBuild()
```

Comment:
Can we just implement this in the tests?

```
bool Test::isMinBuild()
{
   vector<Ice::PluginFactory> pluginFactories;
   addDefaultPluginFactories(pluginFactories);
   return any_of(
      pluginFactories.begin(),
      pluginFactories.end(), 
      [](const auto& factory) { 
         return factory.pluginName == "IceUDP" || factory.pluginName == "IceWS"; })
}
```

Reply:
`addDefaultPluginFactories` is an internal, non-exported function. I find a separate exported function cleaner.

Reply:
Ok, I didn't notice we were not exporting addDefaultPluginFactories